docs/self-hosting/govern/plane-ai/configure-embedding-model.md

```
@@ -0,0 +1,151 @@
+  ]
+}
+```
+
+Replace `<your-region>` with your `BR_AWS_REGION` value.
+:::
+
+:::info AWS managed OpenSearch
+Auto-deploy requires direct ML access to OpenSearch. AWS managed OpenSearch restricts this, so deploy the model manually using the [AWS OpenSearch embedding guide](/self-hosting/govern/aws-opensearch-embedding), then use the model ID option above.
+:::
+
+##  Restart Plane
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)